_layouts/services.html

```diff
@@ -184,6 +193,11 @@
       margin-right: 1em;
       margin-left: 1em;
     }
+    img.service-logo {
+      height: 4em;
```

Comment:
If you want to be more specific with logo size, set width and height and specify `object-fit: contain;`